pagopa/io-app
# chore: [IOBP-334] Add `productPagoPA` icon to pagoPA notice pay action in barcode scan screen

⚠️ Depends on https://github.com/pagopa/io-app-design-system/pull/128

## Short description
This PR adds the `productPagoPA` icon to pagoPA notice pay action in barcode scan screen.

<img src="https://github.com/pagopa/io-app/assets/6160324/0333ee83-2353-4cef-afce-400a49ef0bab" width="200" />

## List of changes proposed in this pull request
- [ts/features/barcode/screens/BarcodeScanScreen.tsx](https://github.com/pagopa/io-app/compare/master...IOBP-334-add-pagopa-icon-in-barcode-scan-bottom-sheet#diff-12570b34029e534a64b24f1cb81383d99557273c4b2f34ad4ced82685fa017f6): added `productPagoPA` icon

## How to test
With the experimental design system enabled, navigate to the barcode scan screen and check that the pagoPA manual input actions displays the correct icon
